quarkus-workshop-super-heroes/docs/src/docs/asciidoc/0-introduction/introduction-installing-ai.adoc

```diff
@@ -0,0 +1,68 @@
+[[introduction-installing-ai]]
+
+= OpenAI or AzureAI Subscriptions
+
+The Narration microservice needs to access an AI service to generate the text narrating the fight.
+You can choose between OpenAI or AzureAI.
```

Comment:
Can we have a small blurb about what this choice means? Why would I choose one or the other?